doc/rst/language/spec/tuples.rst

```diff
@@ -829,6 +829,19 @@ in the two operand tuples. Otherwise, a compile-time error will result.
 Predefined Functions and Methods on Tuples
 ------------------------------------------
 
+.. code-block:: chapel
```

Comment:
If the `.. function` definitions below render nicely, could we just remove these?

Reply:
Since I think you've timed out for the day, and in the interest in getting this in, I'm going to leave this as-is for now and rely on #15097 as a way of getting the spec cleaned up w.r.t. `proc` definitions.

Reply:
Sorry - I glanced at this too quickly and thought it was modifying the doc string of https://chapel-lang.org/docs/builtins/ChapelTuple.html. I am in full agreement, and am happy that you removed the actual code block in the doc 👍